Users/coafile.rst

```diff
@@ -39,6 +39,12 @@ wide coafile which has the lowest priority. The ``default_coafile`` must
 lie in the coala installation directory and is valid for everyone using
 this coala installation.
 
+It can be used to define the type of files you usually don't want to lint,
+like minified files (e.g. ``*.min.js``) and backup files (e.g. ``*.orig``),
+one such example is: ``ignore =
```

Comment:
This is a bad example.  Please remove it.

Reply:
Then which example should i keep?

Reply:
you are mentioning `.min.js`-files and `.orig` files, so why you don't use them? Don't use complicated coala-related paths^^

Reply:
Also use a multiline code block instead of double-backticks^^

Reply:
what is multiline code blocks?

Reply:
There's another syntax in RST to format code blocks, that is designed for code spanning multiple lines. The double-backticks are used for in-line text. E.g. "We have to call `func()` to process stuff."
I think it's using `::`, however please look it up again :+1:

Reply:
btw theres no example of `.min.js` or `.orig` in ignore.